dockers/docker-snmp-sv2/Dockerfile.j2

```diff
@@ -41,7 +41,6 @@ RUN apt-get -y install build-essential wget libssl-dev openssl supervisor && \
     python3 -m sonic_ax_impl install && \
     python3 -m pip uninstall -y pip setuptools && \
     /bin/bash -c "rm -rf /usr/lib/python3.5/{unittest,lib2to3,tkinter,idlelib,email,test}" && \
-    apt-get -y purge build-essential wget libssl-dev openssl && \
```

Comment:
Is that true 'openssl' is the root cause? If yes, still purge others since they will eat up lots of space.